src/components/repeat/repeat.component.tsx

```diff
@@ -25,6 +26,8 @@ const Repeat: React.FC<FormFieldProps> = ({ question, onChange, handler }) => {
   const [counter, setCounter] = useState(0);
   const [rows, setRows] = useState([]);
   const [fieldComponent, setFieldComponent] = useState(null);
+  const [showDeleteModal, setShowDeleteModal] = useState(false);
+  const [selectedQuestion, setSelectedQuestion] = useState(null);
```

Comment:
`questionToDelete` reads better

Reply:
Yep it sounds much better. Updated the names accordingly :D